Makefile

```diff
@@ -1,18 +1,19 @@
 INSTALL=install
 PREFIX=/usr
 SYSCONFDIR=/etc
+PKG_CONFIG ?= pkg-config
```

Comment:
With `?=` if you have `PKG_CONFIG` environment variable its value will be used instead of this one (`pkg-config`).